lib/matplotlib/patheffects.py

```diff
@@ -159,7 +161,35 @@ class Normal(AbstractPathEffect):
     The Normal PathEffect's sole purpose is to draw the original artist with
     no special path effect.
     """
-    pass
+
+
+def _subclass_with_normal(effect_class):
+    """
+    Create a PathEffect class combining *effect_class* and a normal draw.
+    """
+
+    class withEffect(effect_class):
+        def draw_path(self, renderer, gc, tpath, affine, rgbFace):
+            super().draw_path(renderer, gc, tpath, affine, rgbFace)
+            renderer.draw_path(gc, tpath, affine, rgbFace)
+
+    withEffect.__name__ = f"with{effect_class.__name__}"
```

Comment:
So... if you have to do this big song and dance for the docs, is this change really worth it?

Reply:
It's more worth it if we keep adding new patheffects and repeat this code pattern again and again, e.g. https://github.com/matplotlib/matplotlib/pull/15458.